Lombiq.Walkthroughs/Assets/Scripts/shepherd-to-window.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable */
+import Shepherd from '/Lombiq.Walkthroughs/shepherd.js/js/shepherd.mjs';
+
+window.Shepherd = Shepherd;
```

Comment:
If I copy this over the parameter of the jquery method will be compromised as it is a param name with the same name as the import. Also it would in my opinion hurt integrity as we shouldn't make the Jquery Walkthrough.js a module just for one import so it does everything. But the first point is stronger.

Reply:
I just copied over and it worked. Don't know if I'm missing something: 
![image](https://github.com/Lombiq/Orchard-Walkthroughs/assets/80963259/4afe45bb-47a9-45fd-bf63-ac6a15d02b01)

Reply:
But okay, since in the future, we might want to add more JS files including shepherd.

Reply:
Yes it's possible that it is working but you have to eslint-disable like 2-3 warnings which here is a bit unnecessary so I opted out of using this. I did think about this it really could go either way tbh.